services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceController.java

```diff
@@ -202,6 +202,7 @@ private void createServices() {
           Optional.of(new StoreBackupVersionCleanupService((VeniceHelixAdmin) admin, multiClusterConfigs));
       LOGGER.info("StoreBackupVersionCleanupService is enabled");
     }
+    initializeSystemSchema(controllerService.getVeniceHelixAdmin());
```

Comment:
Is this code move corresponding to the following part in the description? If so, can we add some short comments before this line to remind future readers?

> Before helix manager initialization, controller builds controller client using url or d2 configs and registers system schemas. Since the controller instance is still disabled in helix, helix won't hand over leadership back to A and won't transit B from leader to standby.

Reply:
Yes. Added a comment.